fastify.js

```diff
@@ -114,9 +121,7 @@ function build (options) {
     [kReply]: Reply.buildReply(Reply),
     [kRequest]: Request.buildRequest(Request),
     [kMiddlewares]: [],
-    [kCanSetNotFoundHandler]: true,
-    [kFourOhFourLevelInstance]: null,
-    [kFourOhFourContext]: null,
+    [kFourOhFour]: fourOhFour,
```

Comment:
It doesn't look like setting this property on the Fastify instance is necessary. The `fourOhFour` object can be accessed anywhere in this closure, so accessing it through `this[kFourOhFour]` is unnecessary.

Reply:
The `fourOfFour` object can't be reached in the `override()` function:
https://github.com/fastify/fastify/pull/1547/files#diff-20da0b6140a4768d182af0d9b185ac28R695

Should I move the override inside the `build()`?
I have tried and the tests are passing. I'm not sure if there are other effects

Reply:
You're right, the `override()` function is outside of the closure so `fourOhFour` can't be reached there. In that case, I'd say it's fine to leave it how it is.